app/imports/components/Users/AdminPanel/UsersManagement/UsersList/index.js

```diff
@@ -0,0 +1,16 @@
+import { Meteor } from 'meteor/meteor';
+import { composeWithTracker } from 'react-komposer';
+import UsersList from './usersList.jsx';
+
+const composer = (props, onData) => {
+    const handler = Meteor.subscribe('userList');
+
+    if (handler.ready()) {
+        const users = Meteor.users.find({});
```

Comment:
Fetch this data and send to the component only array. Separate `Meteor` things from `React`. Now you are sending `Mongo.Cursor` to the component.